source/Calamari.Shared/Integration/Scripting/ScriptSyntax.cs

```diff
@@ -1,16 +1,15 @@
 ﻿using System.Linq;
-using System.Reflection;
 using Calamari.Commands.Support;
 
 namespace Calamari.Integration.Scripting
 {
-    public enum ScriptType
+    public enum ScriptSyntax
```

Comment:
To allow calamari to write the correct script (with the right extension) we are synchronizing the script syntax enum with the server.